[MRG] Added 'l2' as acceptable input for loss in l1_min_c()

maxcopeland opened the pull request:
Fixes #11851

Currently, passing `"l2"` to `loss` parameter raises `ValueError` in `l1_min_c()`.
Added `"l2"` as acceptable input for `loss`.

maxcopeland committed:
Adding 'l2' as acceptable input for loss in l1_min_c()

agramfort commented:
are you reverting this commit https://github.com/scikit-learn/scikit-learn/blob/f6fd03efccafe7bc5c7bbae80f5946f1d1ead042/sklearn/svm/bounds.py#L55 ?

amueller commented:
I think the desired fix would have been to remove it from the error message.

maxcopeland committed:
removed "l2" from error msg in l1_min_c()

removed `"l2"` from error msg in l1_min_c(), consistent with acceptable `loss` params

maxcopeland commented:
Ah, I see-- that also explains why the build failed. Sorry about that. I removed "l2" from error message.

amueller commented:
thanks
